app/src/db/migrations/20220627000000_002-metadata-tags.js

```diff
@@ -0,0 +1,122 @@
+const stamps = require('../stamps');
+
+exports.up = function (knex) {
+  return Promise.resolve()
+
+    // // create metadata table
+    .then(() => knex.schema.createTable('metadata', table => {
+      table.specificType(
+        'id',
+        'integer GENERATED ALWAYS AS IDENTITY PRIMARY KEY'
+      );
+      table.string('key').index();
+      table.string('value').index();
```

Comment:
Even though Knex defaults to 255, we should generally be explicit about length declarations, especially in migration files. This is because in the future, if Knex decides to change its default, we would have just introduced a very hard to track down bug.
```js
table.string('key', 255).index();
table.string('value', 255).index();
```